frontend/src/admin/UserListModal.js

```diff
@@ -270,18 +253,15 @@ export function UserListModal({
                       defaultValue={editingUserRole}
                       onChange={handleChange}
                     >
-                      {(editingUserRole === 'USER' ||
-                        (permission === 'SUPER_ADMIN' &&
-                          editingUserRole === 'ADMIN')) && (
+                      {(['PENDING', 'USER'].includes(editingUserRole) ||
```

Comment:
I'm getting a bug when changing trying to change a `PENDING` user to `USER` role. Clicking the edit button then confirming with `Role: USER` results in the `updateUserRole` mutation being sent out with `role: "PENDING"`. I think this is just because the default form value is for the user's current role and `PENDING` isn't an option